lib/stage-copy.ts

```diff
@@ -147,6 +147,7 @@ export function resolveFileDestination(
 	dest: string,
 	pathInTar: string,
 ) {
+	// TODO: source.split probably needs to use native separator (i.e. path.sep), but needs investigating.
 	const sourceParts = source.split('/').filter(p => p !== '');
```

Comment:
As per comment, this line probably needs changing.  However I haven't yet been able to trigger this codepath to test.